applications/StructuralMechanicsApplication/custom_elements/membrane_element.cpp

```diff
@@ -1157,21 +1173,20 @@ void MembraneElement::CalculateAndAddBodyForce(VectorType& rRightHandSideVector)
 {
     KRATOS_TRY
     auto& r_geom = GetGeometry();
-    if (r_geom[0].SolutionStepsDataHas(VOLUME_ACCELERATION)){
```

Comment:
why do you remove this?

Reply:
this was not intended .... sry :flight_arrival: